src/test_l1_deepSleepMgr.c

```diff
@@ -238,8 +240,10 @@ void test_l1_deepSleepMgr_negative_PLAT_DS_TERM (void)
     UT_LOG("\n In %s [%02d%03d]\n", __FUNCTION__, gTestGroup, gTestID);
 
     // Variation 01: Attempt to close interface before initialization and check the result
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
     result = PLAT_DS_TERM();
     UT_ASSERT_EQUAL_FATAL(result, DEEPSLEEPMGR_NOT_INITIALIZED);
```

Comment:
For  the enhance error codes NOT_INITIALISED or NOT_TERMINATED, most of the cases still return a different error.  This should be handled.
Handle in whole document